apps/web/lib/getTeamMemberEmailFromCrm.ts

```diff
@@ -19,23 +19,23 @@ interface EventData {
   schedulingType: SchedulingType | null;
   metadata: Prisma.JsonValue | null;
   length: number;
+  hosts: {
+    user: {
+      email: string;
+    };
+  }[];
```

Comment:
Include hosts emails in the initial query here so we don't need to make a second DB query when checking if the CRM record owner is a host.